config/test_dtm0_config.yaml

```diff
@@ -0,0 +1,11 @@
+test_41204:
```

Comment:
are these test specific and if not can they go in common?
Also for log_file_specific, we can directly define in test as well.

Reply:
Done